src/Sylius/Bundle/ApiBundle/Validator/Constraints/UserNotFound.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\Validator\Constraints;
+
+use Symfony\Component\Validator\Constraint;
+
+/** @experimental */
+final class UserNotFound extends Constraint
```

Comment:
It can be used also with the AdminUser that's why I named it User instead of ShopUser.
That make me think about creating service ID that match this fact (see it in the next commits, coming soon).

I agree with the renaming, I had read your issue but not with enough attention I guess...

Reply:
Hmm, I fail to understand how you could have this working with two or more user classes in parallel. With one, yes, you'd override the service definition, but what about two? You would have two validator services, but this constraint will always return the same service id, won't it?

Reply:
Good point @rimas-kudelis I thought about the service definition but not about the way Validators are working....
I'll fix this ;)